Omit extra libraries because their interfaces aren't exposed to applications
by libkrb5.  Discussion on krbdev suggests that this will be controlled one
way or another by the --deps flag in future upstream releases.

--- krb5-1.6/src/krb5-config.in.extralibs	2007-05-15 15:00:15.000000000 -0500
+++ krb5-1.6/src/krb5-config.in	2007-05-15 15:07:04.000000000 -0500
@@ -222,7 +222,7 @@
     fi
 
     if test $library = 'krb5'; then
-	lib_flags="$lib_flags -lkrb5 -lk5crypto -lcom_err $GEN_LIB $LIBS $DL_LIB"
+	lib_flags="$lib_flags -lkrb5 -lk5crypto -lcom_err"
     fi
 
     echo $lib_flags
